### DISCUSSION TITLE
咱们 alova 有 node 环境重试策略吗，看了下文档有前端环境的，是不是还没有 node 环境的

### DISCUSSION THREAD

**Discussion by iamobj** [alovajs/alova#383] [Feature Request]
### What problem does the new feature need to solve?

是不是 node 环境没有内置的重试策略，要自己实现？

### How important is it to you?

Better to have

**Comment by JOU-amjs** [comment]
目前服务端还没有，不过我们3.0中会添加服务端的请求策略，包括重试策略，期待一下哦
